### DISCUSSION TITLE
Is there a lib for mann whitney test and two-way anova?

### DISCUSSION THREAD

**Discussion by chris-bec** [stdlib-js/stdlib#444] [Q&A]
this library looks so great, but I'm going to use mann whitney test and two-way anova, is there a proper lib for them here? thanks a lot

**Comment by chris-bec** [comment]
add another one: Friedman test

**Comment by Planeshifter** [comment]
Hey there,

thanks for the kind words and your interest in the project. Unfortunately, we currently do not have implementations for these three tests. 

The available tests can all be found in the [`stats` namespace](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/stats). 

Pull requests with contributions are always welcome, but we could also look into implementing these over the next months.
